spec/views/accounts/connected_accounts/selected_email/edit.html.erb_spec.rb

```diff
@@ -21,6 +21,7 @@
     @identity = identity
     @select_email_form = SelectEmailForm.new(user:, identity:)
     @can_add_email = true
+    @email_id = user.email_addresses.first.id
```

Comment:
I don't think we need to assign this anymore after caf53d26645483bd5ecbd13df5e2ee6215532685 ? Since the controller action we're simulating no longer assigns it.

```suggestion
```